test/focusable.jest.test.js

```diff
@@ -0,0 +1,275 @@
+const { focusable } = require('../src/index.js');
+const {
+  basic,
+  nested,
+  jqueryui,
+  'non-linear': nonLinear,
+  'changing-content': changingContent,
+  svg,
+  radio,
+  details,
+  'shadow-dom': shadowDom,
+} = require('./fixtures/index.js');
```

Comment:
For clarity in the test code, could we import this as `const fix = require(...)` and then have, for example, `fix.jqueryui` in the code when we're using a fixture? I think that would help with readability. Same thing for `test/tabbable.jest.test.js` where we also import the fixtures module.

Reply:
Will do!